_ext/events.rb

```diff
@@ -0,0 +1,50 @@
+require 'aweplug/helpers/searchisko'
+require 'parallel'
+
+module JBoss
+  module Developer
+    class Events
+
+      def initialize
+
+      end
+
+      def execute site
+        searchisko = Aweplug::Helpers::Searchisko.default site, 0
+
+        Parallel.each(site.events, in_threads: (site.build_threads || 0)) do |(id, data)|
+
+        searchisko_hash = data.collect { |(key, value)|
+
+          case key
+            when 'title'
+              ['sys_title', value]
+            else
+              [key, value]
+          end
+        }.to_h
+
+        searchisko_hash.merge!({:unique_id => searchisko_hash['sys_title'].downcase.strip.gsub(' ', '-').gsub(/[^\w-]/, '') + "-#{id}"})
+
+        searchisko_hash.merge!({:sys_url_view => "#{site.base_url}/events/event#!id=#{searchisko_hash[:unique_id]}"})
+
+        ## Set a default priority if none set. 100 is lowest possible priority value.
```

Comment:
You don't need priority here. That's just on the connectors page to apply a bespoke ordering.